src/main/java/com/gbtec/GenerationPattern.java

```diff
@@ -0,0 +1,35 @@
+package com.gbtec;
+
+public class GenerationPattern {
+
+    public GenerationPattern() {
+
+    }
+
+    public static void createGlider(Generation generation) {
+        generation.getGeneration().get(0).get(0).setStatus(true);
```

Comment:
instead of using ".get().get()" you can create a method in the "Generation"-class. This method should take the x-  and y-Position of the cell and should return the cell.

At the end this lines should looks like "generation.getCell(0, 0).setStatus(true)"

If you also add the methods "killCell()" and "createCell()" the line could looks like:
"generation.getCell(0, 0).createCell()"